skafka/src/main/scala/com/evolutiongaming/skafka/consumer/ConsumerConfig.scala

```diff
@@ -109,7 +110,8 @@ object ConsumerConfig {
     checkCrcs: Boolean,
     interceptorClasses: List[String],
     excludeInternalTopics: Boolean,
-    isolationLevel: IsolationLevel
+    isolationLevel: IsolationLevel,
+    saslSupport: SaslSupportConfig,
```

Comment:
could you please add another constructor for backward compatibility

Reply:
Done